danger-kotlin-kts/build.gradle

```diff
@@ -18,14 +18,14 @@ apply from: file('../dependencyVersions.gradle')
 apply from: file('version.gradle')
 
 dependencies {
-    provided "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
-    provided "org.jetbrains.kotlin:kotlin-script-runtime:$versionKotlin"
-    provided "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.1.0"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    implementation "org.jetbrains.kotlin:kotlin-script-runtime:$versionKotlin"
+    implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:1.1.0"
     includeJar "org.jetbrains.kotlin:kotlin-main-kts:$versionKotlin"
 }
 
 shadowJar {
-    baseName = 'danger-kotlin-kts'
+    getArchiveBaseName().set('danger-kotlin-kts')
```

Comment:
I didn't set the other properties to `''` here because it doesn't look like anything is depending on the version not being in the filename.